### ISSUE TITLE
Pythagorean Forest: Orange hangs on opening widget

### ISSUE THREAD

**Issue by wvdvegte** [biolab/orange3#6651]
**What's wrong?**
When trying to open Pythagorean Forest, Orange becomes unresponsive and has to be force-quit.

**How can we reproduce the problem?** 

[Pythagorean Forest Bug.ows.zip](https://github.com/biolab/orange3/files/13440251/Pythagorean.Forest.Bug.ows.zip)
The problem seems dependent on system configuration or environment. I have seen Pythagorean Forest open without problems in the same workflow on other systems.

**What's your environment?**

- Operating system: Mac OS 14.1 on Silicon
- Orange version: 3.36.2
- How you installed Orange: from DMG

**Comment by ajdapretnar** [comment]
Can confirm. Mac OS 12.2.1 and Orange 3.36.1 and latest.
Exits with abort trap.
Also, the following error appears:
```
----------------------------- TypeError Exception -----------------------------
Traceback (most recent call last):
  File "/Users/ajda/orange/orange3/Orange/widgets/visualize/owpythagoreanforest.py", line 189, in paint
    offset = option.rect.topLeft() + QPointF(offset_w, offset_h)
             ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for +: 'QPoint' and 'QPointF'
-------------------------------------------------------------------------------
```

**Comment by ajdapretnar** [comment]
@wvdvegte Would you mind writing to orange@biolab.si? We'd like to send you a small gift as a token of appreciation for your valuable suggestions. :)

**Comment by janezd** [comment]
Fixed via #6654.
